src/utilities/introspectionFromSchema.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import invariant from '../jsutils/invariant';
+import { getIntrospectionQuery } from './introspectionQuery';
+import { GraphQLSchema } from '../type/schema';
+import { execute } from '../execution/execute';
+import { parse } from '../language/parser';
+import type {
+  IntrospectionQuery,
+  IntrospectionOptions,
+} from './introspectionQuery';
+
+/**
+ * Build an IntrospectionQuery from a GraphQLSchema
+ *
+ * IntrospectionQuery is useful for utilities that care about type and field
+ * relationships, but do not need to traverse through those relationships.
+ *
+ * This is the inverse of buildClientSchema. The primary use case is outside
+ * of the server context, for instance when doing schema comparisons.
+ */
+export function introspectionFromSchema(
+  schema: GraphQLSchema,
+  options: IntrospectionOptions,
+): IntrospectionQuery {
```

Comment:
Just a note, it seems like the response type of this should be named `IntrospectionResult`

Reply:
@leebyron It's pretty confusing name but it already defined here:
https://github.com/graphql/graphql-js/blob/863fe2b9bfac689904f3640e912730766cf8a809/src/utilities/introspectionQuery.js#L117
Would be great to rename it to `IntrospectionResult` but I'm not sure how much code it will brake 😞